### DISCUSSION TITLE
Desafios do básico ao avançado

### DISCUSSION THREAD

**Discussion by andrelsa** [frontendbr/forum#1869] [Discussão]
Olá pessoal :smile: 

Minha esposa esta iniciando a carreira como desenvolvedora, migrando de área, Analista Financeira para dev frontend, é o que ela mais se identificou.

Ela já esta estudando por conta própria a uns 2 meses, iniciará o curso de Análise e Desenvolvimento de sistemas a partir do próximo semestre. Mas ela não esta tão satisfeita apenas com as aulas que ela assiste, assinou a Alura.

Ontem ela me perguntou onde poderia conseguir desafios para frontend do básico ao avançado, quer implementar o que vem aprendendo nas aulas. Provavelmente tenha alguma issue referente a este tópico, mas não consegui achar no fórum. :confused: 

Caso alguém possa indicar desafios, agradeço.

Obrigado a todos :smile:

**Comment by igorthurow** [comment]
Oi @andrelsa! 

Vocês conhecem o https://www.freecodecamp.org/? 
Acho bacana para aprender o CSS Flexbox o https://flexboxfroggy.com/
E CSS Grid em https://cssgridgarden.com/

Além desses sites que são mais interativos, indico reprodução de layout pra praticar as técnicas aprendidas :)

Sucesso!

**Comment by nathpaiva** [comment]
@andrelsa, chama ela para vir participar do fórum, vai ser bem bacana conhece-la.

Acho bem bacana o texto traduzido pelo @felipefialho, [Do Zero a Herói Front-End](https://www.felipefialho.com/blog/do-zero-a-heroi-do-front-end-parte-1/), além disso, tem o bootcamp do [front-end masters](https://frontendmasters.com/bootcamp/), sem contar a plataforma em sim, tem muito conteúdo bom.

**Comment by felipefialho** [comment]
@andrelsa Esses são desafios um pouco mais complexos já que são testes reais usados pelas empresas, mas é interessante tentar praticar com eles quando ela tiver um pouco mais de experiência:

- [Front-end Challenges](https://github.com/felipefialho/frontend-challenges)

**Comment by andrelsa** [comment]
Obrigado pessoal, pela ajuda, as dicas já repassei a ela, vou falar pra ela participar tb do fórum. :smile:

**Comment by ninetails** [comment]
Se não me engano, assinando o Alura creio que também ela tenha acesso ao fórum de lá. É legal que ela pode também trocar experiências com quem também está acompanhando o curso.

<sup><sub>Mas aqui é mais legal, temos bolo ![Kappa](http://static-cdn.jtvnw.net/emoticons/v1/25/1.0)</sub></sup>
